src/os/bluestore/BlueStore.cc

```diff
@@ -16551,6 +16572,11 @@ int BlueStore::_omap_rmkeys(TransContext *txc,
 
  out:
   dout(10) << __func__ << " " << c->cid << " " << o->oid << " = " << r << dendl;
+  c->store->log_latency(
```

Comment:
I am not sure about value of capturing latency here. We are only measuring time spent on construction a transaction.
It is in `submit` that the actual cost of this op shows up.

Reply:
you're totally right. Removing.